packages/main/src/Avatar.ts

```diff
@@ -350,12 +351,20 @@ class Avatar extends UI5Element implements ITabbable {
 		this._checkInitials();
 	}
 
+	onEnterDOM() {
+		ResizeHandler.register(this, this._resizeHandler.bind(this));
```

Comment:
if we register a resize handler on onEnterDOM it's good to deregister it in onExitDOM.